### ISSUE TITLE
Error Type: <type 'exceptions.ValueError'>

### ISSUE THREAD

**Issue by ratmole** [nixgates/plugin.video.seren#73]
https://github.com/nixgates/plugin.video.seren/blob/90285e7abcb8cb35267f032050a17ad8884ac561/resources/lib/common/tools.py#L327

when setting custom view types and one of them is blank,
i get: Error Type: <type 'exceptions.ValueError'>

with this code it works as intended: 
```
try:
        viewType = int(viewType)
    except:
        viewType = 'Default'
        pass
```

**Comment by nixgates** [comment]
Thanks I'll set that now, looks like I overlooked that. Added in for next public update

**Comment by nixgates** [comment]
Resolved in 0.2
